### ISSUE TITLE
Stream Scaling resolution list missing 1920x1080

### ISSUE THREAD

**Issue by roman380** [RainwayApp/bug-tracker#345]
# Title

Stream Scaling resolution list is missing 1920x1080 option.

Monitor's resolution is 3440x1440. Choices exist but the one that would make most sense is not there.

![](https://cdn.discordapp.com/attachments/359715202989686785/594173902289240069/unknown.png)

(#support around 2019-06-28 15:00 UTC)

## What is expected

1920x1080 should be there no matter what.

**Comment by roman380** [comment]
Workaround - steps to apply custom resolution via edit of settings:

1. from Rainway tray icon menu open Dev Utils, Open Cache Folder
2. the will be a folder opened, before doing anything else exit from Rainway from the same tray icon menu
3. in the opened folder edit settings.json file with notepad (open with... Notepad or otherwise)
4. closer to the end of the file you will see lines where you  can substitute numbers 1920, 1080
    "ScaleWidth": 1920,
    "ScaleHeight": 1080,
5. save the edit and launch Rainway again - it will start with scaling enabled at this new resolution

**Comment by AetherCollective** [comment]
If I recall correctly, it hides the current resolution. Did you select the correct monitor (the 4k one)

**Comment by roman380** [comment]
Yeah it excludes the current selection from the choices dropped down. I, for instance, can't repro on my dev machine with native 3840x2160. However the user's screenshot above suggests that his current setting is "Disabled" so we'd expect 1920x1080 to be available for selection. His complaint in first place was that he was unable to do 1920x1080.

**Comment by agentd00nut** [comment]
Maybe instead of hiding the resolution we could make it visible but visually separate it from the other items in the list to show that its the currently selected item.

Little less confusing and gives a slight hint that a user might not have the proper monitor selected.

**Comment by AetherCollective** [comment]
Maybe like an asterisk to the left of the currently selected resolution?
